### PR TITLE
Update url format for omnibus collection pages (artblocks, prohibition)

### PR DESCRIPTION
**Pull request by kaitoo1**
### Summary of Changes
Currently the collection page url for omnibus pages (Artblocks and Prohibition) differ from other collection page urls by not containing the "/community/" prefix. This makes it a bit more complex to support deep links to the mobile app. This PR adds the community prefix for consistency.

Before: https://gallery.so/artblocks/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a/0
After: https://gallery.so/community/artblocks/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a/0

I also added a redirect on the nft detail page route handler to redirect to the new collection page url if it detects via the query params that the user is visiting the old Artblocks or Prohibition url. (the old url format fits the nft detail page pattern so it gets handled by the nft detail page route)

### Demo or Before/After Pics

Before (artblocks)
CleanShot 2024-02-27 at 20 29 10@2x

After (artblocks)
CleanShot 2024-02-27 at 20 27 04@2x

Before (Prohibition)
CleanShot 2024-02-27 at 20 28 57@2x

After (prohibition)
CleanShot 2024-02-27 at 20 28 41@2x

Redirect in action

https://github.com/gallery-so/gallery/assets/80802871/66faee1c-0e76-4620-b213-9c0e3ebbcccb

### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

general test
- go to any prohibition or artblocks collection page. (examples above)
- the new url should start with /community/

entry points
- different entry points to the collection page should point to the right url
- ie Search result, Collection Pill, Collection Name on NFT Hover

redirect
- go to the old url (without "/community/")
- you should be redirect to the new url
### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/web/pages/[username]/[collectionId]/[tokenId]/index.tsx [hunk 1]
```diff
@@ -43,6 +43,20 @@ export default function NftDetailPage({ username, collectionId, tokenId }: NftDe
...
+  if ((username === 'artblocks' || username === 'prohibition') && collectionId.startsWith('0x')) {
+    return (
+      <GalleryRedirect
+        to={{
+          pathname: `/community/${username}/[contractAddress]/[projectId]`,
+          query: { contractAddress: collectionId, projectId: tokenId },
+        }}
+      />
+    );
+  }
```
Comment (Robinnnnn) [inline apps/web/pages/[username]/[collectionId]/[tokenId]/index.tsx:58]: nit: this should be configurable in next.config.js ([our file](https://github.com/gallery-so/gallery/blob/main/apps/web/next.config.js#L88)), but you'll need to do some regex trickery

tbh i don't think it's worth spending the time on given we probably won't even hit this redirect very often

**Review by Robinnnnn — APPROVED** [review]
